src/common/components/signUpModal/signUpModal.tpl.html

```diff
@@ -121,6 +121,9 @@
          ng-click="$ctrl.signUp()"
          translate>Sign Up</a>
     </div>
+    <p class="text-center">
+      <span>Already have an account?</span> <a href="" ng-click="$ctrl.onStateChange({state: 'sign-in'})" translate=""><span class="ng-scope">Sign In</span></a>
```

Comment:
Looks like you copied the html from the browser which rendered the angular directives. Can you change `translate=""` to `translate` and toss the span with `ng-scope`? Thanks